src/admin-views/events-meta-box/date-time.php

```diff
@@ -0,0 +1,111 @@
+<table cellspacing="0" cellpadding="0" id="EventInfo">
+	<tr>
+		<td colspan="2" class="tribe_sectionheader">
+			<div class="tribe_sectionheader" style="">
+				<h4><?php esc_html_e( 'Time &amp; Date', 'the-events-calendar' ); ?></h4></div>
+		</td>
+	</tr>
+	<tr>
+		<td colspan="2">
+			<table class="eventtable">
+				<tr id="recurrence-changed-row">
+					<td colspan='2'><?php printf( esc_html__( 'You have changed the recurrence rules of this %1$s.  Saving the %1$s will update all future %2$s.  If you did not mean to change all %2$s, then please refresh the page.', 'the-events-calendar' ), $events_label_singular_lowercase, $events_label_plural_lowercase ); ?></td>
```

Comment:
Using esc_html__ here only escapes the translation string, not the singular/plural labels. You ought to wrap those in their own esc_html(), or run `echo esc_html( sprintf( .... ) );` instead.